repo2docker/buildpacks/base.py

```diff
@@ -643,7 +643,12 @@ def get_post_build_scripts(self):
         return []
 
     def get_start_script(self):
-        start = self.binder_path('./start')
+        start = self.binder_path('start')
         if os.path.exists(start):
-            return start
+            # Return an absolute path to start
+            # This is important when built container images start with
+            # a working directory that is different from ${REPO_DIR}
+            # This isn't a problem with anything else, since start is
+            # the only path evaluated at container start time rather than build time
+            return os.path.join('${REPO_DIR}', start)
```

Comment:
This relies on `$REPO_DIR` being an absolute path right? Is there any benefit to calling `os.path.abspath` to make it absolute-as-it-was-at-build-time? I have no idea :-/

Reply:
@betatim yeah, we can't tell at build time if it's an absolute or relative path, so I don't think it'd be useful.